prov/efa/test/efa_unit_test_ep.c

```diff
@@ -310,6 +311,85 @@ void test_efa_rdm_ep_pkt_pool_page_alignment(struct efa_resource **state)
 	fi_close(&ep->fid);
 }
 
+/**
+ * @brief When using LL128 protocol, test the packet allocated from read_copy_pkt_pool
+ *  is 128 byte aligned.
+ * 
+ * @param[in]	state		struct efa_resource that is managed by the framework
+ */
+void test_efa_rdm_read_copy_pkt_pool_128_alignment(struct efa_resource **state)
+{
+	bool iface_initialized;
+	enum fi_hmem_iface iface;
+	struct efa_rdm_pke *pkt_entry;
+	struct efa_rdm_ep *efa_rdm_ep;
+	struct efa_resource *resource = *state;
+
+	/* Override global attribute */
+	iface = FI_HMEM_CUDA;
+	iface_initialized = hmem_ops[iface].initialized;
+	hmem_ops[iface].initialized = true;
+
+	/* rx_readcopy_pkt_pool is only created when application requested FI_HMEM capability */
+	resource->hints = efa_unit_test_alloc_hints(FI_EP_RDM);
+	assert_non_null(resource->hints);
+	resource->hints->caps |= FI_HMEM;
+	resource->hints->domain_attr->mr_mode |= FI_MR_HMEM | FI_MR_ALLOCATED;
```

Comment:
I realized that `fi_getinfo(...)` with FI_HMEM would fail on non-HMEM platforms, because non of CUDA/NEURON/SYNAPSEAI is available at runtime. You might try configure without `--with-cuda`

It would be easier to initialize resources without FI_HMEM, and modify the attributes directly, which is sad :<

Reply:
See my comment below, I think this test doesn't have to do such cuda/hmem hack